habitat-baselines/habitat_baselines/rl/ppo/policy.py

```diff
@@ -30,6 +31,31 @@
     from omegaconf import DictConfig
 
 
+@dataclass
+class PolicyActionData:
+    """
+    Information returned from the `Policy` class.
```

Comment:
List all of the fields and what they do.  The first 4 are pretty obvious, but `take_actions` and `should_inserts' not so much.
    rnn_hidden_states
    actions
    values
    action_log_probs
    take_actions
    policy_info
    should_inserts

Reply:
Done.